config/features/config.go

```diff
@@ -244,6 +245,7 @@ func ConfigureBeaconChain(ctx *cli.Context) error {
 		logEnabled(disableVerboseSigVerification)
 		cfg.EnableVerboseSigVerification = false
 	}
+	cfg.EnableProposerPreprocessing = ctx.Bool(enableProposerPreprocessing.Name)
```

Comment:
Could this follow the standard pattern of logging that this feature is enabled? It would help us diagnose user issues in logs to see this feature was enabled in the logs.